scripts/aws/main.yaml

```diff
@@ -53,6 +53,14 @@ Parameters:
     Type: String
     Description: OpenSearch nodes type
 
+  OpenSearchMasterUsername:
+    Type: String
+    Description: OpenSearch master username
+
+  OpenSearchMasterPassword:
+    Type: String
+    Description: OpenSearch master password
+
```

Comment:
Passarla come parametro di tipo String la espone in chiaro. Preferirei usare una [dynamic reference](https://docs.aws.amazon.com/AWSCloudFormation/latest/UserGuide/dynamic-references.html)